src/commands/scanner/run.ts

```diff
@@ -137,12 +137,13 @@ export default class Run extends ScannerCommand {
 			output = await ruleManager.runRulesMatchingCriteria(filters, targetPaths, outputOptions, engineOptions);
 		} catch (e) {
 			// Rethrow any errors as SFDX errors.
-			throw new SfdxError(e.message || e, null, null, INTERNAL_ERROR_CODE);
+			const message: string = e instanceof Error ? e.message : e as string;
```

Comment:
For some reason, `e` can't be declared as any type besides `any` or `unknown`. So we're using this snippet to convert `e` into something we definitely know is a string. This appears many times in the PR.